demos/requirements.txt

```diff
@@ -1,13 +1,18 @@
+inflect>=5.3.0
 librosa>=0.8.0
 matplotlib>=3.3.4
 motmetrics>=1.2.0
 nibabel>=3.2.1
 numpy>=1.16.6,<1.20; python_version <= "3.6"
 numpy>=1.16.6,<=1.21; python_version > "3.6"
+opencv-python==4.5.*
+openvino-dev
```

Comment:
currently demos really depends on openvino-dev? If I right remember, it needs only if user wants to use downloader or mo, I can not to say that it is a strong requirement
only `openvino` package required for now

Reply:
Updated